js/libs/canvas2d.js

```diff
@@ -124,6 +189,11 @@ class C2D {
     this.ctx.closePath()
     this.ctx.stroke()
   }
+
+  static text (text, x, y, stroke = false) {
+    if (!stroke) this.ctx.fillText(text, x, y)
+    else this.ctx.strokeText(text, x, y)
+  }
```

Comment:
why is this either or? what if the developer wants to create text with both a stroke and a width?

Reply:
ooo that's a good point — i think that mostly happened because i wanted a single binary variable to handle this (since its clean) and the actual goal of the function followed that instead of a user-oriented approach. in retrospect, looking at it from the user p.o.v. is probably a good idea going forward

went ahead and edited the function by using a case-switch on a string that specifies the desired behavior, and allows for both (and in either order)